.travis.yml

```diff
@@ -308,21 +278,20 @@ script:
   # cc bazel tests (w/o RLlib)
   - ./ci/suppress_output bazel test --config=ci --build_tests_only --show_progress_rate_limit=100 --test_output=errors -- //:all -rllib/...
 
+  - local bazel_targets=()
   # ray serve tests
-  - if [ $RAY_CI_SERVE_AFFECTED == "1" ]; then ./ci/keep_alive bazel test --config=ci --test_tag_filters=-jenkins_only python/ray/serve/...; fi
+  - if [ $RAY_CI_SERVE_AFFECTED == "1" ]; then bazel_targets+=(python/ray/serve/...); fi
 
   # ray operator tests
-  - cd ./deploy/ray-operator/
-  - CC=gcc ../../ci/suppress_output go build
-  - CC=gcc ../../ci/suppress_output go test ./...
-  - cd ../..
+  - (cd deploy/ray-operator && export CC=gcc && suppress_output go build && suppress_output go test ./...)
 
   # random python tests TODO(ekl): these should be moved to bazel
   - if [ $RAY_CI_PYTHON_AFFECTED == "1" ]; then python -m pytest -v --durations=5 --timeout=300 python/ray/experimental/test/async_test.py; fi
 
   # bazel python tests. This should be run last to keep its logs at the end of travis logs.
-  - if [ $RAY_CI_PYTHON_AFFECTED == "1" ]; then ./ci/keep_alive bazel test --config=ci --test_tag_filters=-jenkins_only python/ray/tests/...; fi
-  - if [ $RAY_CI_TUNE_AFFECTED == "1" ]; then ./ci/keep_alive bazel test --config=ci --test_tag_filters=-jenkins_only python/ray/tune/...; fi
+  - if [ $RAY_CI_PYTHON_AFFECTED == "1" ]; then bazel_targets+=(python/ray/tests/...); fi
```

Comment:
Why is this change needed?

Reply:
It factors out and deduplicates the `bazel test` calls for maintainability.

Reply:
No, it makes it harder to read (and harder to copy and paste into a terminal if you want to run it.

Reply:
Hmm okie